config/mbed_app.json

```diff
@@ -15,6 +15,7 @@
 			"bluenrg_ms.SPI_nCS": "BLE_SPI_NSS",
 			"bluenrg_ms.SPI_SCK": "BLE_SPI_SCK",
 			"cordio.desired-att-mtu": 251,
+			"cordio.max-prepared-writes": 1,
```

Comment:
@YannLocatelli can you explain why the default value of `4` was not working and why the value of `1` is better? 

my understanding is that a lower value might impact the efficiency of operations requiring multiple attribute writes.

is `1` used to avoid more than one thing happening at once? and to make sure each command are processed one at a time?

can you explain the issue you were encountering?

Reply:
I talked about it few months ago, with value of 4 it works but we have to handle it differently.

You receive 4 packets at the same time in the robot and since you write directly in the file the data you received, you have to handle writing + ordering at the same time
I tried some solutions but never succeed, if you have any idea, let me know.